### ISSUE TITLE
Best practice to make DMTCP script available to Python

### ISSUE THREAD

**Issue by mathiasbockwoldt** [dmtcp/dmtcp#880]
I know that there is a [python script](https://github.com/dmtcp/dmtcp/blob/master/contrib/python/dmtcp.py) that allows control over DMTCP. But how should I put it in the Python PATH? Should I copy the script to some place Python can find it? Is that automagically done for me during installation?
Did you consider turning the script into a module on [PyPi](https://pypi.org/) so that users can install it, e.g., using `pip install dmtcp`?

**Comment by watefeenex** [comment]
Hey @mathiasbockwoldt I'm currently trying to use DMTCP on windows and I agree with you that it would have been easier as a PyPi module but this is a Linux package by the looks of it. I just wanted to know, were you able to make it work? I got Ubuntu 18.04 LTS cause that is the version it is compatible, and built it. However I've run into the problem below.
![image](https://user-images.githubusercontent.com/49417921/116488882-a12e2600-a8e7-11eb-8e5b-e6edb8cdd2dd.png)
I have no clue where the shared area is.

**Comment by mathiasbockwoldt** [comment]
No, sorry. I tried various things, but in the end, I did not manage to get DMTCP running with my application anyway, so the Python controls became irrelevant.
Good luck!

**Comment by watefeenex** [comment]
@mathiasbockwoldt By any chance do you know if dmtcp can checkpoint is created during native code (i.e. numpy/scipy/sklearn routines). Also so in the end did you try and do checkpointing in python? If so how did you manage to do it?

**Comment by mathiasbockwoldt** [comment]
I did not manage to get checkpointing running. Neither with "native" DMTCP nor with Python.
